webapp/src/app/ui/avatar/avatar-config.ts

```diff
@@ -0,0 +1,24 @@
+import { inject, InjectionToken, Provider } from '@angular/core';
+
+export interface AvatarConfig {
+  delayMs: number;
+}
+
+export const defaultAvatarConfig: AvatarConfig = {
+  delayMs: 0
```

Comment:
Is `0ms` the best default value for us? I remember Shadcn recommending `600ms` to accommodate for slower connections, but unfortunately can't find any references for it right now. The official RadixUI example uses `600`, although the library has a default of `0` (https://www.radix-ui.com/primitives/docs/components/avatar).

Reply:
Yeah I was also thinking the same, I also saw the 600 once. I think I will adjust it to prevent flickering